package.json

```diff
@@ -132,13 +133,15 @@
     "preset": "react-native",
     "roots": [
       "<rootDir>/node_modules/react-native/",
-      "<rootDir>/lib/",
+      "<rootDir>/src/",
       "<rootDir>/test/react/",
+      "<rootDir>/test/",
       "<rootDir>/fb-www/"
     ],
     "testEnvironment": "jsdom",
     "testMatch": [
       "**/test/react/*-test.js",
+      "**/test/*-test.js",
```

Comment:
Can we replace two lines above with just `'**/test/**/*-test.js`? Or would it not include the root `/test/*-test` files?